examples/programmatic-highlighting/assets/styles.css

```diff
@@ -0,0 +1,26 @@
+html,
+body {
+  font-family: "Heebo";
+  transition: background-color 0.5s ease;
+}
+
+.body-content {
+  display: flex;
+}
+.chart {
+  flex: 1;
+  width: 50%;
+  height: 70%;
+}
+.options {
+  display: flex;
+  justify-content: center;
+}
+
+.select {
+  height: 30px;
+}
```

Comment:
The styling is pretty weird - the dropdown's height makes it not look like a dropdown. Can we change the layout so that the select appears above the query textbox?

Reply:
Change made in bcaf65f